website/data/intro-navigation.js

```diff
@@ -10,6 +10,33 @@ export default [
   { category: 'vs', content: ['ecs', 'mesos', 'terraform'] },
   {
     category: 'getting-started',
-    content: ['running', 'jobs', 'cluster', 'ui', 'next-steps'],
+    name: 'Getting Started',
+    content: [
```

Comment:
For what it's worth, the intro page is no longer linked to from anywhere (though it's still in search results).

Reply:
Yeah, we can definitely remove it if the team feels like that's the right move. I just didn't feel like that was my choice to make 😅